src/server/req_runjob.c

```diff
@@ -432,21 +432,18 @@ req_runjob(struct batch_request *preq)
 	/* resources, then process the run request, else it has    */
 	/* to go to the scheduler				   */
 
-	if ((preq->rq_ind.rq_run.rq_destin == NULL)  ||
-		(*preq->rq_ind.rq_run.rq_destin == '\0')) {
-		char fixjid[PBS_MAXSVRJOBID+1];
+	if ((preq->rq_ind.rq_run.rq_destin == NULL) ||
+	    (*preq->rq_ind.rq_run.rq_destin == '\0')) {
+		char fixjid[PBS_MAXSVRJOBID + 1];
 
 		/* if runjob request is from the Scheduler, */
 		/* it must have a destination specified     */
 		if (preq->rq_conn == psched->scheduler_sock) {
-			sprintf(log_buffer,
-				"runjob request from scheduler with null destination");
-			log_event(PBSEVENT_SYSTEM, PBS_EVENTCLASS_JOB, LOG_INFO,
-				jid, log_buffer);
+			log_event(PBSEVENT_SYSTEM, PBS_EVENTCLASS_JOB, LOG_INFO, jid, "runjob request from scheduler with null destination");
```

Comment:
Since you've been making other changes to bring this code in line with our standard, consider fixing the multiple single line comments to being one multiline comment.  Up to you.